projects/storefrontlib/layout/launch-dialog/config/launch-config.ts

```diff
@@ -90,4 +90,5 @@ export const enum LAUNCH_CALLER {
   ANONYMOUS_CONSENT = 'ANONYMOUS_CONSENT',
   REPLENISHMENT_ORDER = 'REPLENISHMENT_ORDER',
   PLACE_ORDER_SPINNER = 'PLACE_ORDER_SPINNER',
+  CLEAR_CART = 'CLEAR_CART',
```

Comment:
I'm thinkinig that perhaps this value shoould be encapsulated iin the cart lib somehow.  Let's discuss options